config/locales/client.en.yml

```
@@ -5217,6 +5217,8 @@ en:
 
         reports:
           trend_title: "%{percent} change. Currently %{current}, was %{prev} in previous period."
+          percent_change_tooltip: "%{percent} change."
+          percent_change_tooltip_previous_value: "Was %{previousValue} %{previousPeriod}."
```

Ideally, you have multiple strings instead of a single string because these concatenated strings are really difficult to translate. Also, there are still hard codded strings in code. And just to be safe, I'd pluralize the strings by using `%{count}` instead of `%{previousValue}`. Previous value is a number, not a string, right?

```suggestion
          percent_change_tooltip_previous_value:
            yesterday:
              one: "Was %{count} two days ago."
              other: "Was %{count} two days ago."
            two_weeks_ago:
              one: "Was %{count} two weeks ago."
              other: "Was %{count} two weeks ago."
            thirty_days_ago:
              one: "Was %{count} in the previous 30 day period."
              other: "Was %{count} in the previous 30 day period."
```